### PR TITLE
Organize script edit

### PR DESCRIPTION
**Pull request by dmcavoy**
Organizes the script edit page by:
1. Using the HelpTip component so we don't have to clutter the page with help text
2. Grouping the fields together into related groups

Checked with the curriculum team and they were on board this change: https://codedotorg.slack.com/archives/CFGAVL2CA/p1589913488188600

Screen Shot 2020-05-19 at 2 30 30 PM
screencapture-localhost-studio-code-org-3000-s-csd3-2019-edit-2020-05-19-14_48_52

## Testing story

All existing tests should cover the behavior since this is just a re organizing of the page.

# Reviewer Checklist:

- [ ] Tests provide adequate coverage
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: apps/src/lib/script-editor/LessonExtrasEditor.jsx [hunk 1]
```diff
@@ -35,20 +36,21 @@ export default class LessonExtrasEditor extends React.Component {
...
             checked={this.state.lessonExtras}
             style={styles.checkbox}
             onChange={e => this.setState({lessonExtras: e.target.checked})}
           />
-          <p>
-            If also enabled by the teacher, show the lesson extras page at the
-            end of each lesson.
-          </p>
+          <HelpTip>
+            <p>
```
Comment (davidsbailey) [inline apps/src/lib/script-editor/LessonExtrasEditor.jsx:49]: just curious if it looks any different/worse without the `<p>`, like this: https://github.com/code-dot-org/code-dot-org/blob/35718de683cbf67fa2761c63bf0f2db3b54fcd44/apps/src/lib/ui/HelpTip.story.jsx#L10

**Review by davidsbailey — APPROVED** [review]
looks great, @dmcavoy !

**Review by uponthesun — APPROVED** [review]
I love this! Looks like a great improvement, and great bang for buck.

File: apps/src/lib/script-editor/LessonExtrasEditor.jsx (hunk 1 above)
Comment (dmcavoy) [reply apps/src/lib/script-editor/LessonExtrasEditor.jsx:49]: I checked and it seems to work fine without the paragraph tag as long as all the text is supposed to be smushed together (not in 2 paragraphs or something). I'm going to leave it for now.

**Pull request merged**